packages/network-controller/src/NetworkController.ts

```diff
@@ -1231,6 +1231,25 @@ export class NetworkController extends BaseControllerV2<
     });
   }
 
+  /**
+   * Searches for a network configuration ID with the given ChainID and returns it.
+   *
+   * @param chainId - ChainId to search for
+   * @returns networkClientId of the network configuration with the given chainId
+   */
+  findNetworkClientIdByChainId(chainId: Hex): NetworkClientId {
```

Comment:
Wondering if it would be more future proof for the us to return all the networkClients for the chainId and let the consumer decide what to do if there are more than one?

Reply:
Ok fair enough!